### DISCUSSION TITLE
How do I enable a feature flag?

### DISCUSSION THREAD

**Discussion by rljacobson** [evcxr/evcxr#179] [Q&A]
In a normal Rust program, I would have (for example) the following at the top of `main.rs`:

```rust
#![feature(unboxed_closures)]
#![feature(fn_traits)] 

// other stuff...
```

Can I achieve the same thing with evcxr?

**Comment by thomcc** [comment]
I don't think so, but I've wanted it too.

**Comment by davidlattimore** [comment]
Good suggestion. I'm just working on adding support for this now...

**Reply by alexdelorenzo** [reply]
This would be a great feature!

**Comment by davidlattimore** [comment]
I guess mentioning discussions from commits doesn't ping the discussion. I [implemented this](https://github.com/google/evcxr/commit/6bf63c0d84b8f5bc0d7cecff940a46e0ad71ff25) a couple of weeks ago. It's not released to crates.io yet, but if you'd like to try it out, you can install from github by doing:

```bash
cargo install --force --git https://github.com/google/evcxr.git evcxr_repl
```

or for the jupyter kernel:

```bash
cargo install --force --git https://github.com/google/evcxr.git evcxr_jupyter
```
